package-lock.json

```diff
@@ -2184,18 +2187,615 @@
         "path-is-absolute": "^1.0.0",
         "readdirp": "^2.2.1",
         "upath": "^1.1.1"
+      },
+      "dependencies": {
+        "fsevents": {
```

Comment:
I don't think we should be checking in fsevents. It's unrelated to the fix and is required only on MacOS, right?

Reply:
Reverted Thanks @apb7